src/OrchardCore.Modules/OrchardCore.Lists/Indexes/ListPartContentIndexHandler.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement;
+using OrchardCore.Indexing;
+using OrchardCore.Lists.Models;
+
+namespace OrchardCore.Lists.Indexes
+{
+    public class ListPartContentIndexHandler : IContentItemIndexHandler
```

Comment:
Why isn't it a `ContentPartIndexHandler<ContainedPart>` then ?

Reply:
True and easier 😄

Reply:
Strange but it's not working now with these changes.

Reply:
Reverting the changes to my previous solution. A ContainedPart is a dynamic welded part which can't be listed from the ContentDefinitionManager.